### PR TITLE
Update blue-green deployment and canary release wording and images

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/project-user-guide/grayscale-release/canary-release.md [hunk 1]
```diff
@@ -1,53 +1,81 @@
...
-![](/images/service-mesh/canary-release-0.png)
+## Prerequisites
 
-## Before you begin
-
-Firstly, you shoud enable the service-mesh plugin, to make the istio component available by following the document [Requirements](../../../pluggable-components/service-mesh/).
+- You need to enable [KubeSphere Service Mesh](../../../pluggable-components/service-mesh/).
+- You need to create a workspace, a project and an account (`project-regular`). Please refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project) if they are not ready yet.
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+- You need to enable **Application Governance** and have an available app so that you can implement the canary release for it. The sample app used in this tutorial is Bookinfo. For more information, see [Deploy Bookinfo and Manage Traffic](../../../quick-start/deploy-bookinfo-to-k8s/).
```
Comment (zackzhangkai) [inline content/en/docs/project-user-guide/grayscale-release/canary-release.md:21]: Actually, this is no need. It's a switch for tracing.

File: content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md [hunk 2]
```diff
@@ -1,51 +1,82 @@
...
 
 
-## Before you begin
+## Prerequisites
 
-Firstly, you shoud enable the service-mesh plugin, to make the istio component available by following the document [Requirements](../../../pluggable-components/service-mesh/).
+- You need to enable [KubeSphere Service Mesh](../../../pluggable-components/service-mesh/).
+- You need to create a workspace, a project and an account (`project-regular`). Please refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project) if they are not ready yet.
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+- You need to enable **Application Governance** and have an available app so that you can implement the blue-green deployment for it. The sample app used in this tutorial is Bookinfo. For more information, see [Deploy Bookinfo and Manage Traffic](../../../quick-start/deploy-bookinfo-to-k8s/).
```
Comment (zackzhangkai) [inline content/en/docs/project-user-guide/grayscale-release/blue-green-deployment.md:21]: **Application Governance**  is just a switch for Tracing. May be no need here.

**Comment by Sherlock113** [comment]
@zackzhangkai I don't think that is the case. If you do not enable application governance, the button is not clickable. You can try that yourself.

截屏2020-11-04下午4 34 46

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**